vue/router.py

```diff
@@ -7,10 +7,20 @@ class VueRouter(Wrapper):
     def init_dict(cls):
         return VueRouterFactory.get_item(cls)
 
-    def __new__(cls):
-        return window.VueRouter.new(cls.init_dict())
+    def __new__(cls, vue_router_class=window.VueRouter):
```

Comment:
This is the last request for a change I have:
Can we revert this change, and not have it in this PR?

I like this feature and would like to have it, but lets keep the scope of this PR only to adding support for the rest of the route params.

I would propose adding this in a second PR, and also take care of proper tests of this feature in the second PR